src/bindings/c/README.md

```diff
@@ -0,0 +1,28 @@
+# OpenVINO C API
+
+OpenVINO C API is a part of OpenVINO Runtime library. 
+This component is responsible for:
+ * Based representations for the objects in OpenVINO - C API provides structures with C code for basical classes inside the OpenVINO, such as "ov_core_t" for "ov::Core". For more information please read [OpenVINO Runtime C API User Guide](./docs/OpenVINO_Runtime_C_API_User_Guide.md).
```

Comment:
```suggestion
 * Based representations for the objects in OpenVINO - C API provides structures with C code for basical classes inside the OpenVINO, such as `ov_core_t` for `ov::Core`. For more information please read [OpenVINO Runtime C API User Guide](./docs/OpenVINO_Runtime_C_API_User_Guide.md).
```

User guides are located on the site. Here we should have only developer documentation